exporter/googlecloudexporter/README.md

```diff
@@ -0,0 +1,60 @@
+# Google Cloud Exporter
+
+This exporter can be used to send metrics, traces, and logs to Google Cloud Monitoring. It is an extension of the official 
+(Google Cloud Exporter)[https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/googlecloudexporter], with additional processors built in to streamline configuration.
+
+## Configuration
+| Field               | Default               | Required | Description |
+| ---                 | ---                   | ---      | ---         |
+| `project`           |                       | `false`  | The GCP project used when exporting telemetry data. |
+| `endpoint`          |                       | `false`  | The endpoint used when exporting telemetry data. |
+| `location`          | `us-east1-b`          | `false`  | The GCP location attribute attached to telemetry data. |
+| `namespace`         | `{hostname}`          | `false`  | The GCP namespace attribute attached to telemetry data. |
+| `user_agent`        | `observIQ-otel-agent` | `false`  | Overrides the user agent used when making requests. |
+| `timeout`           | `12s`                 | `false`  | The timeout for API calls. |
+| `resource_mappings` | [See below](#resource-mapping-default)         | `false`  | Defines a mapping of resources from source to target. |
+| `retry_on_failure`  |                       | `false`  | Handle retries when sending data to Google Cloud fails. |
+| `sending_queue`     |                       | `false`  | Determines how telemetry data is buffered before exporting. |
+| `batch`             |                       | `false`  | The config of the exporter's [batch processor](https://github.com/open-telemetry/opentelemetry-collector/tree/main/processor/batchprocessor). |
```

Comment:
This is a bit of a pain but it's probably worth making sure this documentation points towards the documentation of the version of these processors that we support. With how fast OTel changes we could have documentation that isn't correct for what we have packaged.

Reply:
Yeah, I believe that's a worthwhile change. The only issue will be we'll have to remember to update this documentation every time we up the version.

Reply:
Yeah, I agree this is the same issue we have with the current docs. Ideally if they're all on the same version it's easy to find replace the entire repo but I doubt that's the case. Might be worth trying to create tooling to generate these.